client/src/main/resources/templates/js/main.js

```diff
@@ -463,27 +463,27 @@ function displaySource(src, title) {
       }
     });
 
-    $(pageAdditionalId + 'testcase-pagination ul .page-number').click(function() {
-      setIndex(pageAdditionalId,((parseInt($(this).attr('data-index')) - 1) * getPageSize(pageAdditionalId)) + 1);
+    $(pageAdditionalId + "testcase-pagination ul .page-number").click(function() {
+      setIndex(pageAdditionalId,((parseInt($(this).attr("data-index")) - 1) * getPageSize(pageAdditionalId)) + 1);
       renderTable(pageAdditionalId, getFilterSearch(pageAdditionalId), getIndex(pageAdditionalId),
         getPageSize(pageAdditionalId));
     });
 
-    $(pageAdditionalId + 'testcase-pagination ul .page-next').click(function() {
+    $(pageAdditionalId + "testcase-pagination ul .page-next").click(function() {
       if(getFilterSearch(pageAdditionalId).length >= (getIndex(pageAdditionalId) + getPageSize(pageAdditionalId))) {
         setIndex(pageAdditionalId, (getIndex(pageAdditionalId) + getPageSize(pageAdditionalId)));
-        var k = $(pageAdditionalId + 'testcase-pagination ul .page-number').last().find('a').text();
+        var k = $(pageAdditionalId + "testcase-pagination ul .page-number").last().find("a").text();
         var n = parseInt(k) + 1;
         if(n <= getNoOfPage(pageAdditionalId)) {
-          $(pageAdditionalId + 'testcase-pagination ul .page-number').first().remove();
-          $(pageAdditionalId + "testcase-pagination ul .page-next").before($('<li>').attr('data-index',n).addClass('page-number').append($('<a>').attr('href','#').append(n) ));
+          $(pageAdditionalId + "testcase-pagination ul .page-number").first().remove();
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Line is too long.](https://www.codacy.com/app/SeLion/paypal-SeLion/file/1307322591/issues/source?bid=2928425&fileBranchId=2951451#l479)